tests/secrets_api_mock/requirements.txt

```diff
@@ -0,0 +1,7 @@
+click==7.1.2
```

Comment:
Is it supposed to be installed in the docker image?

Reply:
No, we don't need it for now.